processor/deltatocumulativeprocessor/internal/metrics/data.go

```diff
@@ -0,0 +1,56 @@
+package metrics
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor/internal/data"
+)
+
+type Data[D data.Point[D]] interface {
+	At(i int) D
+	Len() int
+	Ident() Ident
+}
+
+type Sum Metric
+
+func (s Sum) At(i int) data.Number {
+	dp := Metric(s).Sum().DataPoints().At(i)
+	return data.Number{NumberDataPoint: dp}
+}
+
+func (s Sum) Len() int {
+	return Metric(s).Sum().DataPoints().Len()
+}
+
+func (s Sum) Ident() Ident {
```

Comment:
What's the point of having an `Ident` method that's identical to the `Identity` method? Could `Data`'s `Ident` method be called `Identity` instead? It's confusing to have two similarly named methods doing the same thing.

Reply:
standardized on `Ident` in all places